index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio Setup</title>
+</head>
+<body>
+     <main>
+        <nav class="menu-icon">
+            <h4 class="logo"><a href="#">My Logo</a></h4>
```

Comment:
- [ ] Kindly make sure your logo has a link tag (`anchor`) tag as it is the convention. and also for semantic reasons and user experience sake. You may check [here](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.
```suggestion
            <a href="#" class="logo">My Logo</a>
```
- [ ] Also make sure you stick to the original design 
### Your design :point_down: 
![image](https://github.com/AbuTalha3/Portfolio-Setup/assets/97100091/3d5c2a34-8572-4798-848f-80c618bfd45c)
### The original template :point_down: 
![image](https://github.com/AbuTalha3/Portfolio-Setup/assets/97100091/41b14738-619f-4318-99e0-12302abd83e9)